java/code/src/com/redhat/rhn/taskomatic/task/AutoErrataTask.java

```diff
@@ -121,10 +162,16 @@ public void execute(JobExecutionContext context)
          * where rtb.name = 'repo-sync-bunch'
          *   and rtr.status in ('RUNNING','READY')
          */
+        if (sids.isEmpty()) {
+            return new ArrayList<>();
+        }
+
+        Map<String, String> params = new HashMap<>();
+        params.put("system_ids", "{" + sids + "}");
```

Comment:
I think this does not look good. Check `systemCurrencyList()` for an alternative way.